plugins/webdriver/nbactions-release-profile.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like this file is something not needed. Can you remove it?

Reply:
Thanks, will remove that file, also, just found out that failed UTs are due to deprecation of methods and properties in Selenium 3.12.0. Will fix UTs.